src/coreComponents/codingUtilities/Utilities.hpp

```diff
@@ -144,32 +144,32 @@ void copy( localIndex const N, VEC1 const & v1, VEC2 const & v2 )
 template< typename MATRIX, typename VEC1, typename VEC2 >
 GEOSX_HOST_DEVICE
 GEOSX_FORCE_INLINE
-void applyChainRule( localIndex const N,
-                     MATRIX const & dy_dx,
-                     VEC1 const & df_dy,
-                     VEC2 && df_dx )
+void applyChainRule( localIndex const n,
+                     MATRIX const & dyDx,
```

Comment:
This has to be kept the way it was. `camelCase` is for spoken word, but the `_` signifies some sort of operator or break in the phrase. In this case, it is `dy/dx` but we can't write it like that. We could do `dy_div_dx` but that is a little too verbose and really isn't what is happening anyways. So the use of `dy_dx` indicates that there is some sort of operator between the terms....leaving the reader to figure that out...which I think is reasonable.

My objection applies to every other replacement of derivative terms.